endpoint/ethProcessorAware.go

```diff
@@ -197,3 +197,9 @@ func (e *EthProcessorAware) GasPrice(ctx context.Context) (*common.Uint256, erro
 		return e.Eth.GasPrice(ctx)
 	})
 }
+
+func (e *EthProcessorAware) FeeHistory(ctx context.Context, blockCount uint, newestBlock common.BN64, percentiles *[]uint) (*response.FeeHistory, error) {
```

Comment:
Let's use `common.Uint64` for `blockCount` here?

Two reasons:
1. Not sure if `uint` will handle hex representation correctly
2. To conform with other methods (`GetTransactionByBlockNumberAndIndex` for example)